File: Numbers/card_validator/card_validator.py
card = input("Enter the card number: ")
list = card.split()
sum = 0

try:
  for num in list:
    num = int(num)
    if num * 2 > 9:
      sum += num * 2 - 9
    else:
      sum += num * 2

  if num % 10 == 0:
    print("That is a valid card number!")
  else:
    print("Invalid card number!")
except ValueError:
  print("Please enter a card number")
  

File: Numbers/card_validator/card_validator.go
package main

import (
	"fmt"
  "strings"
  "strconv"
)

func main() {
  var sum int
	fmt.Println("Enter the card number: ")
  var card string
  fmt.Scan(&card)
  arr := strings.Split(card, "")
  for i := range arr {
    num, err := strconv.Atoi(arr[i])
    if err != nil {
      println("Please enter numbers only")
    }
    if num * 2 > 9 {
      sum += num * 2 - 9
    } else {
      sum += num * 2
    }
  }
  if sum / 10 == 0 {
    println("Valid card number")
  } else {
    println("Invalid card number")
  }
}
